index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <html lang="en">
+<head>
+  <title>Car Lot | Single Page Application Quiz</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+</head>
+<body>
+
+<div class="container">
+<!-- Start of Nav Bar -->
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+    <h1>Janelle's Used Car Inventory</h1>
+  <!-- Start of Input Form -->
+  <form class="form-inline" onsubmit = "return false">
+    <div class="form-group">
+      <label for="text-field">Enter New Description:</label>
+      <input type="text" class="form-control" id="text-field">
+    </div>
+    <!-- Button that does nothing -->
+    <button class="btn btn-default">Change Description</button>
+  </form>
+
+    </div>
+  </div>
+</nav>
+<!-- Inventory Area -->
+<div id="car-garage" class="row"></div>
```

Comment:
This works for now, and meets the requirements, but keep in mind that your code is limited to working with three car cards only. What would happen if your business grew and you had more than three cars? Just a brain teaser to think about for future implementations of dynamic injection of DOM elements.